components/wasm-opt/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::new_without_default)]
```

Comment:
I figured this warning would be more annoying than helpful since the `new` method on the types it was complaining about don't take args.